ocaml/lambda/translprim.ml

```diff
@@ -1450,7 +1454,8 @@ let lambda_primitive_needs_event_after = function
   | Punboxed_float_array_set_128 _ | Punboxed_int32_array_set_128 _
   | Punboxed_int64_array_set_128 _ | Punboxed_nativeint_array_set_128 _
   | Prunstack | Pperform | Preperform | Presume
-  | Pbbswap _ | Pobj_dup | Pget_header _ -> true
+  | Pbbswap _ | Pobj_dup | Pget_header _
+  | Preinterpret_tagged_int63_as_unboxed_int64 -> true
```

Comment:
This is because unboxing is ignored in bytecode, right?
Would be good to update the comment at the top of this function.

Reply:
Yeah I think I've made a mistake here looking at Chris's comments, will check - but yes, that is the intention.